Ray.jl/src/object_store.jl

```diff
@@ -5,13 +5,15 @@ Store `data` in the object store. Returns an object reference which can used to
 the `data` with [`Ray.get`](@ref).
 """
 function put(data)
+    buffer = _put(data)
+    return ray_jll.put(buffer)
+end
+
+function _put(data)
     bytes = Vector{UInt8}()
     io = IOBuffer(bytes; write=true)
     serialize(io, data)
-    buffer_ptr = Ptr{Nothing}(pointer(bytes))
-    buffer_size = sizeof(bytes)
-    buffer = rayjll.LocalMemoryBuffer(buffer_ptr, buffer_size, true)
-    return rayjll.put(buffer)
+    return ray_jll.LocalMemoryBuffer(bytes, sizeof(bytes), true)
 end
```

Comment:
I sorta feel like this should be a constructor for `LocalMemoryBuffer` (or `to_local_memory_buffer`) in the jll

Reply:
The problem is the `serialize` component. The `ray_jll` module doesn't care about serialization at all and you can put any data within a `ray_jll.LocalMemoryBuffer`. Maybe I'll just make this function `to_serialized_buffer`.

Reply:
yeah. on the other hand, there's really only one thing you can put in a local memory buffer and that's a vector of bytes, and serialization seems like a good fallback for how to turn something into bytes...

Reply:
I'm not too bothered about where this goes though as long as the name is clearer